src/haddock/modules/refinement/emref/__init__.py

```diff
@@ -1,4 +1,13 @@
-"""Energy minimization refinement with CNS."""
+"""Energy minimization refinement with CNS.
+
+The ``[emref]`` module refine the input complexes by energy minimization using
+the conjugate gradient method developed by
+`Powell <https://en.wikipedia.org/wiki/Powell%27s_method>`_,
```

Comment:
Seems random to add a link to Powel and not other things. I would  remove it and simply write: "by conjugate gradient energy minimisation."